tests/test_enhanced_sampling.py

```diff
@@ -120,6 +121,45 @@ def align_sample(x_gas, x_solvent):
     return xb_new
 
 
+from typing import List
+from md.states import CoordsVelBox
+
+
+class AlignedMover(MultipleTryMetropolis):
+    def __init__(self, K, vacuum_samples, vacuum_log_weights, log_prob_fn):
+
+        self.vacuum_samples = vacuum_samples
+        self.vacuum_log_weights = vacuum_log_weights
+        self.batch_log_prob_fn = jax.vmap(log_prob_fn)
+
+        super().__init__(K)
+
+    def batch_propose(self, x: CoordsVelBox) -> List[CoordsVelBox]:
```

Comment:
Docstring should note assumption that ligand is in the last `num_ligand_atoms` indices of `x` -- in future, we might want to replace coordinates of a different subsystem.

Reply:
willfix